package.json

```diff
@@ -1,66 +1,111 @@
 {
-  "name": "victory",
-  "version": "0.27.2",
+  "name": "victory-monorepo",
+  "version": "0.0.0",
   "description": "Data viz for React",
   "keywords": [
     "data visualization",
     "React",
     "d3",
     "charting"
   ],
-  "main": "lib/index.js",
-  "module": "es/index.js",
-  "jsnext:main": "es/index.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/formidablelabs/victory.git"
   },
-  "engines": {
-    "npm": ">=2.0.0"
-  },
+  "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
   "author": "Formidable",
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/formidablelabs/victory/issues"
   },
-  "homepage": "https://github.com/formidablelabs/victory",
+  "homepage": "https://github.com/formidablelabs/open-source/victory",
   "scripts": {
-    "postinstall": "builder run npm:postinstall",
-    "preversion": "builder run npm:preversion",
-    "postversion": "builder run npm:postversion",
-    "postpublish": "builder run npm:postpublish",
-    "start": "builder run dev",
-    "test": "builder run check",
-    "version": "builder run npm:version"
-  },
-  "dependencies": {
-    "builder": "^4.0.0",
-    "builder-victory-component": "^5.1.2",
-    "victory-chart": "^27.1.0",
-    "victory-core": "^24.0.0",
-    "victory-pie": "^16.1.0"
+    "postinstall": "lerna exec --parallel -- nps build-libs",
+    "preversion": "nps check",
+    "start": "nps start",
+    "storybook": "start-storybook -p 6006",
+    "chromatic": "chromatic test --storybook-addon",
+    "chromatic-ci": "chromatic test --storybook-addon --exit-zero-on-changes"
   },
   "devDependencies": {
```

Comment:
Have you tried a git npm install like:

```sh
$ npm install FormidableLabs/victory#pre-1.0.0
```

?

When you do that you actually shouldn't get **any** devDependencies, so the `postinstall` should fail. Worth considering what our "git dependency" story is now that we're going monorepo, including dropping support all together.

/cc @tptee